messaging-client-databus/src/main/java/com/inmobi/messaging/consumer/databus/DatabusConsumer.java

```diff
@@ -88,6 +85,9 @@
   public static String clusterNamePrefix = "databusCluster";
   private Boolean readFromLocalStream;
   private int numList = 0;
+  public static List<String> collectorsWithReaders = new ArrayList<String>();
+  private Timer partitionDiscovererAndReaderCreator;
+  private Long MINUTE = 1000l;
```

Comment:
Can we make it static final constant and change value to **60 * 1000** instead **1000**?

private static final Long MINUTE to NUB_OF_SECS_IN_MINUTE =  60 * 1000 ;